src/java/org/apache/cassandra/db/MultiCBuilder.java

```diff
@@ -418,6 +448,35 @@ public MultiCBuilder addAllElementsToAll(List<List<ByteBuffer>> values)
             return set.build();
         }
 
+        @Override
+        public List<ByteBuffer> buildSerializedPartitionKeys()
+        {
+            built = true;
+
+            if (hasMissingElements)
+                return Collections.EMPTY_LIST;
+
+            TreeSet<ByteBuffer> set = comparator.size() == 1 ? new TreeSet<>(comparator.subtype(0))
```

Comment:
If the point of the `TreeSet` is to remove the equal entries, could you add a comment saying that? Otherwise, it would be easier to send directly to the array list and then sort.

Reply:
This method is used to return a unique sorted list of partition key values that data is being requested for. I've added a comment to reflect this.